lib/Typst.ml

```diff
@@ -36,7 +39,11 @@ let rec inline_to_typst buf = function
       raise
         (Markdown_to_typst_mismatch
            "HTML content cannot be easily converted to Typst markup.")
-  | _ -> raise Unimplemented
+  | Emph _ -> ()
```

Comment:
while this is already an improvement over the exception previously thrown, could you at least add the respective contents to the buffer, or - even better - add it with formatting? i.e.

```ocaml
| Emph ( attr, inl ) -> Buffer.add_string buf "_"; inline_to_typst buf inl; Buffer.add_string buf "_"
```

etc.